dace/properties.py

```diff
@@ -329,7 +329,7 @@ def initialize_properties(obj, *args, **kwargs):
         for name, prop in own_properties.items():
             # Only assign our own properties, so we don't overwrite what's been
             # set by the base class
-            if hasattr(obj, name):
+            if hasattr(obj, '_' + name):
```

Comment:
Shouldn't we check for both?

Reply:
The reason it is not checked for is that it will return True if you override it as a `@property`, even if the type was not stored.